GiniComponents/GiniUtilites/Sources/GiniUtilites/GiniLocalization.swift

```diff
@@ -20,69 +20,69 @@ public enum GiniLocalization: String, CaseIterable {
  A utility for retrieving localized strings from the client's bundle or SDK bundle.
  */
 enum GiniLocalized {
-    
+
     /**
      Retrieves a localized string for the given key. According localization GiniHealthConfiguration localization field and with check for client app locallizaton
-     
+
      - Parameters:
      - key: The key to search for in the strings file.
      - fallbackKey: The fallback key to use if the primary key is not found.
      - comment: The corresponding comment for the key.
-     
+
      - Returns: The localized string for the given key.
      */
-    static func string(_ key: String, fallbackKey: String? = nil, comment: String) -> String {
-        let locale = GiniHealthConfiguration.shared.customLocalization?.rawValue
+    static func string(_ key: String, fallbackKey: String? = nil, comment: String, locale: String, bundle: Bundle) -> String {
```

Comment:
[Clarification]: We had a property in Health for the forced localisation `GiniHealthConfiguration.shared.customLocalization`
How do we handle it now?

Reply:
Actually we don't. I need to find a solution to this. Max refactored everything into GiniLocalized but it's not used anywhere